package.json

```diff
@@ -77,39 +78,32 @@
     ],
     "contributes": {
         "viewsContainers": {
-            "activitybar": [
```

Comment:
Revert the whitespace changes in this file. It should be auto-formatted by VS Code when you save, but you can also manually run "Format document" by right clicking in the file
![Screen Shot 2020-02-04 at 10 28 08 AM](https://user-images.githubusercontent.com/11282622/73774580-1472d780-4739-11ea-9d7e-fab74a7a3618.png)